src/components/cafes/[id]/OriginList/index.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image';
+import brazil from '@/assets/Icon/brazil.svg';
+import { orginItem } from './OriginList.css';
+
+interface OriginProps {
+  origins: { id: number; origin: string }[];
+}
+
+const OriginList = ({ origins }: OriginProps) => {
+  return (
+    <ul>
+      {origins.map((item) => (
+        <li key={item.id} className={orginItem}>
+          <Image src={brazil} alt={`${brazil} 아이콘`} />
```

Comment:
이걸 의도하신 게 맞을까요~? 추가로 brazil까지 한국어로 적어줘도 좋을 것 같아요!
```suggestion
          <Image src={brazil} alt="brazil 아이콘"} />
```